docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -104,7 +104,7 @@ def generate_input_port_for_remote_process_group(remote_process_group, name):
         return input_port_node
 
     def add_test_data(self, path, test_data, file_name=str(uuid.uuid4())):
-        self.docker_directory_bindings.put_file_to_docker_path(self.test_id, path, file_name, test_data.encode('utf-8'))
+        self.docker_directory_bindings.put_file_to_docker_path(self.test_id, path, file_name, test_data.replace('\\n', '\n').encode('utf-8'))
```

Comment:
note: I have also touched this part for the same reason (use escaped `\t`, `\n`, ... chars) in [1168](https://github.com/apache/nifi-minifi-cpp/pull/1168) (same with the file content validation)

Reply:
OK, I'll revert these bits if your PR is merged first.